client/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-
+import { Route } from 'react-router-dom';
```

Comment:
How you liking this pattern?
When should we use { var } from ... and when should we use var from ... ?